src/EmployerPR/SFA.DAS.EmployerPR/Application/Queries/GetAccountLegalEntities/GetAccountLegalEntitiesQueryHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using MediatR;
+using SFA.DAS.EmployerPR.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerPR.Application.Queries.GetAccountLegalEntities;
+public class GetAccountLegalEntitiesQueryHandler : IRequestHandler<GetAccountLegalEntitiesQuery, GetAccountLegalEntitiesQueryResult>
+{
+    private readonly IAccountsApiClient<AccountsConfiguration> _apiClient;
+
+    public GetAccountLegalEntitiesQueryHandler(IAccountsApiClient<AccountsConfiguration> apiClient)
+    {
+        _apiClient = apiClient;
+    }
+    public async Task<GetAccountLegalEntitiesQueryResult> Handle(GetAccountLegalEntitiesQuery request, CancellationToken cancellationToken)
+    {
+        var accountLegalEntities =
```

Comment:
This response should not be nullable, it can be empty though

Reply:
fixed